frontend/src/api/backendClient.ts

```diff
@@ -7,6 +7,15 @@ export interface User {
   aboutMe?: string
 }
 
+export interface UserLeaderBoard {
+  userName: string
+  totalPointsEarned: number
+}
+
+export interface UserLeaderBoards {
```

Comment:
UserLeaderBoards could not be renamed to LeaderBoard since when we import this in the LeaderBoard.tsx page - we have an error importing it. So are keeping this as UserLeaderBoard

Reply:
that's a good solution